### PR TITLE
Add libxmlb & appstream

### PR DESCRIPTION
**Pull request by der-eismann**
Stumbled over them a few times in the GNOME universe, might as well add them when I have already built them.

### CODE REVIEW

File: a/appstream/stone.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - pkgconfig(libcurl)
+    - pkgconfig(libsystemd)
+    - pkgconfig(libxml-2.0)
+    - pkgconfig(libzstd)
+    - pkgconfig(xmlb)
+    - pkgconfig(yaml-0.1)
+toolchain   : gnu
+setup       : |
+    sed -i '/missing-include-dirs/d' meson.build
+    %meson -Dstemming=false -Dvapi=true -Dsvg-support=false -Dapidocs=false -Dinstall-docs=false --sysconfdir=/usr/share/appstream
```
Comment (ermo) [inline a/appstream/stone.yml:41]: Please use the %(datadir) definition (defined here: https://github.com/serpent-os/boulder/blob/main/data/macros/arch/base.yml#L11)

**Review by ermo — CHANGES_REQUESTED** [review]
Small nit, otherwise looks good.

File: a/appstream/stone.yml (hunk 1 above)
Comment (der-eismann) [reply a/appstream/stone.yml]: Fixed!

**Review by ermo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**